sw/host/tpm2_test_server/src/main.rs

```diff
@@ -59,9 +67,11 @@ pub fn main() -> std::io::Result<()> {
         .filter(None, options.logging)
         .init();
     let cmd_addr = SocketAddr::from(([127, 0, 0, 1], options.tpm_port));
-    let platform_addr = SocketAddr::from(([127, 0, 0, 1], options.tpm_port + 1));
     let cmd_listener = TcpListener::bind(cmd_addr)?;
+    let port = cmd_listener.local_addr()?.port();
+    let platform_addr = SocketAddr::from(([127, 0, 0, 1], port + 1));
```

Comment:
Is is necessary that the two ports are consecutive?  If so, do we need to account for the possibility that the `port + 1` could be in use by chance, and in that case start over with a different port for `cmd_listener`?